pagerduty/data_source_pagerduty_event_orchestrations.go

```diff
@@ -0,0 +1,127 @@
+package pagerduty
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/heimweh/go-pagerduty/pagerduty"
+)
+
+func dataSourcePagerDutyEventOrchestrations() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourcePagerDutyEventOrchestrationsRead,
+
+		Schema: map[string]*schema.Schema{
+			"search": {
```

Comment:
I suggest you change this attribute for `name_filter`, just to be more alined with common practices used by Hashicorp implementation in others Provider like the AWS one, which uses the attribute `filter` on listed data source to provide an interface for supplying search parameters, but in their case `filter` is a key-value structure, whereas in our case We are using exclusively the **name** as the lookup field.